### PR TITLE
Oncall plugin-related UI warnings and improvements

### PR DESCRIPTION
**Pull request by mvilanova**
Screen Shot 2020-12-09 at 10 47 08 AM

### CODE REVIEW

File: src/dispatch/plugin/service.py [hunk 1]
```diff
@@ -85,6 +88,15 @@ def update(*, db_session, plugin: Plugin, plugin_in: PluginUpdate) -> Plugin:
                     f"Cannot disable plugin: {plugin.title}. It is required and no other plugins of type {plugin.type} are enabled."
                 )
 
+        if plugin.type == OncallPlugin.type:
+            oncall_services = service_service.get_all_by_type_and_status(
+                db_session=db_session, service_type=plugin.slug, is_active=True
+            )
+            if oncall_services:
+                raise InvalidConfiguration(
+                    f"Cannot disable plugin: {plugin.title}. One or more oncall services depend on it. "
```
Comment (kevgliss) [inline src/dispatch/plugin/service.py:97]: Should we include which services are dependent so the user doesn't have to go and find them?

**Review by kevgliss — APPROVED** [review]

File: src/dispatch/plugin/service.py (hunk 1 above)
Comment (mvilanova) [reply src/dispatch/plugin/service.py:97]: I thought about it, but decided not to for two reasons: 1) the error message could get very long if there are a lot of services that depend on it 2) the error message disappears after a few seconds. Instead I make it easier for the user by adding a column for enabled or not.

Comment (kevgliss) [reply src/dispatch/plugin/service.py:97]: We could always make the error message last longer (or until dismissed).

Comment (mvilanova) [reply src/dispatch/plugin/service.py:97]: I think we should do it regardless. I find it annoying that it disappears so quickly.

**Pull request merged**